IPython/terminal/ptshell.py

```diff
@@ -232,7 +230,10 @@ def _(event):
         self._style = self._make_style_from_name(self.highlighting_style)
         style = DynamicStyle(lambda: self._style)
 
+        editing_mode = EditingMode.VI if self.vi_mode else EditingMode.EMACS
```

Comment:
In the future, possibly yes, but not in the near future. I wanted to have the flexibility, but I don't have the intention right now to add another set of key bindings. So far, also nobody asked me. Nano/Pico key bindings would be an example.